README.md

```diff
@@ -28,17 +28,21 @@ The `auro-background` element is an easy-to-use mobile-first alternative to writ
 <!-- AURO-GENERATED-CONTENT This file is to be used for any additional content that should be included in the README.md which is specific to this component. -->
 <!-- AURO-GENERATED-CONTENT:END -->
 
-## UI development browser support
+## Use Cases
 
-<!-- AURO-GENERATED-CONTENT:START (REMOTE:url=https://raw.githubusercontent.com/AlaskaAirlines/auro-templates/main/templates/default/partials/browserSupport.md) -->
-For the most up to date information on [UI development browser support](https://auro.alaskaair.com/support/browsersSupport)
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./docs/partials/useCases.md) -->
```

Comment:
This "Use Cases" section needs to be updated --- This is not actually describing the use case for this component

Reply:
If you think that info is valuable, it could be added to the `readmeAddlInfo.md` file